widgetssdk/src/main/java/com/glia/widgets/UiTheme.kt

```diff
@@ -639,6 +653,20 @@ data class UiTheme(
         var iconOnHold: Int? = null
             private set
 
+        /**
+         * The icon resID used when visitor is sharing the screen
+         */
+        @DrawableRes
+        var iconEndScreenShare: Int? = null
+        private set
+
+        /**
+         * The icon tint resId when visitor is sharing the screen
```

Comment:
NIT: I see that in most other places you use ID in capital letters, so thought it would be nice to have it here as well.
```suggestion
         * The icon tint resID when visitor is sharing the screen
```